config/routes.rb

```diff
@@ -20,5 +20,7 @@
 
   resources :spots, only: [:index, :show, :create, :new, :edit, :update]
 
+  resources :users, only: [:index, :destroy]
```

Comment:
This is the second time you've listed resources users, with index listed in both. What's the functionality you're looking for here?

Reply:
This is used for an API call (returning spots for a given user) on a branch that hasn't merged yet.